2012/07/23 20:27:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in W:\home\lovehatmy\www\framework\web\CWebApplication.php:280
Stack trace:
#0 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user')
#1 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=user
---
2012/07/23 22:49:32 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in W:\home\lovehatmy\www\framework\db\CDbConnection.php:382
Stack trace:
#0 W:\home\lovehatmy\www\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 W:\home\lovehatmy\www\framework\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 W:\home\lovehatmy\www\framework\base\CModule.php(388): CDbConnection->init()
#3 W:\home\lovehatmy\www\framework\base\CApplication.php(431): CModule->getComponent('db')
#4 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(615): CApplication->getDb()
#5 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(2263): CActiveRecord->getDbConnection()
#6 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(379): CActiveRecordMetaData->__construct(Object(User))
#7 W:\home\lovehatmy\www\protected\models\User.php(13): CActiveRecord::model('User')
#8 W:\home\lovehatmy\www\protected\controllers\UserController.php(9): User::model()
#9 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionIndex()
#10 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#11 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#12 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('')
#14 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user')
#15 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#16 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/index.php?r=user
---
2012/07/23 22:50:07 [error] [exception.CDbException] SQLSTATE[42000] [1049] Unknown database 'testdrive'
2012/07/23 22:50:07 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in W:\home\lovehatmy\www\framework\db\CDbConnection.php:388
Stack trace:
#0 W:\home\lovehatmy\www\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 W:\home\lovehatmy\www\framework\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 W:\home\lovehatmy\www\framework\base\CModule.php(388): CDbConnection->init()
#3 W:\home\lovehatmy\www\framework\base\CApplication.php(431): CModule->getComponent('db')
#4 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(615): CApplication->getDb()
#5 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(2263): CActiveRecord->getDbConnection()
#6 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(379): CActiveRecordMetaData->__construct(Object(User))
#7 W:\home\lovehatmy\www\protected\models\User.php(13): CActiveRecord::model('User')
#8 W:\home\lovehatmy\www\protected\controllers\UserController.php(9): User::model()
#9 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionIndex()
#10 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#11 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#12 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('')
#14 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user')
#15 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#16 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/index.php?r=user
---
2012/07/23 22:50:18 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in W:\home\lovehatmy\www\framework\db\CDbConnection.php:382
Stack trace:
#0 W:\home\lovehatmy\www\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 W:\home\lovehatmy\www\framework\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 W:\home\lovehatmy\www\framework\base\CModule.php(388): CDbConnection->init()
#3 W:\home\lovehatmy\www\framework\base\CApplication.php(431): CModule->getComponent('db')
#4 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(615): CApplication->getDb()
#5 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(2263): CActiveRecord->getDbConnection()
#6 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(379): CActiveRecordMetaData->__construct(Object(User))
#7 W:\home\lovehatmy\www\protected\models\User.php(13): CActiveRecord::model('User')
#8 W:\home\lovehatmy\www\protected\controllers\UserController.php(9): User::model()
#9 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionIndex()
#10 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#11 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#12 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('')
#14 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user')
#15 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#16 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/index.php?r=user
---
2012/07/23 22:51:24 [error] [exception.CException] exception 'CException' with message 'Property "User.password" is not defined.' in W:\home\lovehatmy\www\framework\base\CComponent.php:131
Stack trace:
#0 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(144): CComponent->__get('password')
#1 W:\home\lovehatmy\www\framework\web\helpers\CHtml.php(2044): CActiveRecord->__get('password')
#2 W:\home\lovehatmy\www\framework\web\helpers\CHtml.php(1774): CHtml::resolveValue(Object(User), 'password')
#3 W:\home\lovehatmy\www\framework\web\helpers\CHtml.php(1238): CHtml::activeInputField('password', Object(User), 'password', Array)
#4 W:\home\lovehatmy\www\protected\views\user\form_signup.php(13): CHtml::activePasswordField(Object(User), 'password')
#5 W:\home\lovehatmy\www\framework\web\CBaseController.php(127): require('W:\home\lovehat...')
#6 W:\home\lovehatmy\www\framework\web\CBaseController.php(96): CBaseController->renderInternal('W:\home\lovehat...', Array, true)
#7 W:\home\lovehatmy\www\framework\web\CController.php(870): CBaseController->renderFile('W:\home\lovehat...', Array, true)
#8 W:\home\lovehatmy\www\framework\web\CController.php(783): CController->renderPartial('form_signup', Array, true)
#9 W:\home\lovehatmy\www\protected\controllers\UserController.php(40): CController->render('form_signup', Array)
#10 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#11 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#12 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#13 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#15 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#16 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#17 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user
---
2012/07/23 22:55:04 [error] [exception.CException] exception 'CException' with message 'The "filter" property must be specified with a valid callback.' in W:\home\lovehatmy\www\framework\validators\CFilterValidator.php:46
Stack trace:
#0 W:\home\lovehatmy\www\framework\validators\CValidator.php(197): CFilterValidator->validateAttribute(Object(User), 'email')
#1 W:\home\lovehatmy\www\framework\base\CModel.php(158): CValidator->validate(Object(User), NULL)
#2 W:\home\lovehatmy\www\protected\controllers\UserController.php(28): CModel->validate()
#3 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#4 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#8 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#9 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user/signup
---
2012/07/23 23:02:47 [error] [exception.CException] exception 'CException' with message 'The "filter" property must be specified with a valid callback.' in W:\home\lovehatmy\www\framework\validators\CFilterValidator.php:46
Stack trace:
#0 W:\home\lovehatmy\www\framework\validators\CValidator.php(197): CFilterValidator->validateAttribute(Object(User), 'email')
#1 W:\home\lovehatmy\www\framework\base\CModel.php(158): CValidator->validate(Object(User), NULL)
#2 W:\home\lovehatmy\www\protected\controllers\UserController.php(28): CModel->validate()
#3 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#4 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#8 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#9 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user/signup
---
2012/07/23 23:07:23 [error] [exception.CException] exception 'CException' with message 'The "filter" property must be specified with a valid callback.' in W:\home\lovehatmy\www\framework\validators\CFilterValidator.php:46
Stack trace:
#0 W:\home\lovehatmy\www\framework\validators\CValidator.php(197): CFilterValidator->validateAttribute(Object(User), 'email')
#1 W:\home\lovehatmy\www\framework\base\CModel.php(158): CValidator->validate(Object(User), NULL)
#2 W:\home\lovehatmy\www\protected\controllers\UserController.php(28): CModel->validate()
#3 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#4 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#8 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#9 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user/signup
---
2012/07/23 23:14:11 [error] [exception.CException] exception 'CException' with message 'The "filter" property must be specified with a valid callback.' in W:\home\lovehatmy\www\framework\validators\CFilterValidator.php:46
Stack trace:
#0 W:\home\lovehatmy\www\framework\validators\CValidator.php(197): CFilterValidator->validateAttribute(Object(User), 'email')
#1 W:\home\lovehatmy\www\framework\base\CModel.php(158): CValidator->validate(Object(User), NULL)
#2 W:\home\lovehatmy\www\protected\controllers\UserController.php(28): CModel->validate()
#3 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#4 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#8 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#9 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user/signup
---
2012/07/24 11:49:23 [error] [exception.CException] exception 'CException' with message 'The "filter" property must be specified with a valid callback.' in W:\home\lovehatmy\www\framework\validators\CFilterValidator.php:46
Stack trace:
#0 W:\home\lovehatmy\www\framework\validators\CValidator.php(197): CFilterValidator->validateAttribute(Object(User), 'email')
#1 W:\home\lovehatmy\www\framework\base\CModel.php(158): CValidator->validate(Object(User), NULL)
#2 W:\home\lovehatmy\www\protected\controllers\UserController.php(28): CModel->validate()
#3 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#4 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#8 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#9 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user/signup
---
2012/07/24 11:51:12 [error] [exception.CException] exception 'CException' with message 'The "filter" property must be specified with a valid callback.' in W:\home\lovehatmy\www\framework\validators\CFilterValidator.php:46
Stack trace:
#0 W:\home\lovehatmy\www\framework\validators\CValidator.php(197): CFilterValidator->validateAttribute(Object(User), 'email')
#1 W:\home\lovehatmy\www\framework\base\CModel.php(158): CValidator->validate(Object(User), true)
#2 W:\home\lovehatmy\www\protected\controllers\UserController.php(28): CModel->validate(true, true)
#3 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#4 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#8 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#9 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user/signup
---
2012/07/24 11:51:31 [error] [exception.CException] exception 'CException' with message 'Property "User.dtime_registration" is not defined.' in W:\home\lovehatmy\www\framework\base\CComponent.php:174
Stack trace:
#0 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(160): CComponent->__set('dtime_registrat...', 1343116291)
#1 W:\home\lovehatmy\www\protected\models\User.php(71): CActiveRecord->__set('dtime_registrat...', 1343116291)
#2 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(1008): User->beforeSave()
#3 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(787): CActiveRecord->insert(NULL)
#4 W:\home\lovehatmy\www\protected\controllers\UserController.php(32): CActiveRecord->save(false)
#5 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#6 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#7 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#8 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#10 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#11 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#12 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user/signup
---
2012/07/24 11:54:15 [error] [exception.CException] exception 'CException' with message 'Property "User.register_datatime" is not defined.' in W:\home\lovehatmy\www\framework\base\CComponent.php:174
Stack trace:
#0 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(160): CComponent->__set('register_datati...', 1343116455)
#1 W:\home\lovehatmy\www\protected\models\User.php(71): CActiveRecord->__set('register_datati...', 1343116455)
#2 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(1008): User->beforeSave()
#3 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(787): CActiveRecord->insert(NULL)
#4 W:\home\lovehatmy\www\protected\controllers\UserController.php(32): CActiveRecord->save(false)
#5 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#6 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#7 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#8 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#10 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#11 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#12 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user/signup
---
2012/07/24 11:55:38 [error] [exception.CException] exception 'CException' with message 'Property "User.dtime_registration" is not defined.' in W:\home\lovehatmy\www\framework\base\CComponent.php:131
Stack trace:
#0 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(144): CComponent->__get('dtime_registrat...')
#1 W:\home\lovehatmy\www\protected\views\user\users_list.php(6): CActiveRecord->__get('dtime_registrat...')
#2 W:\home\lovehatmy\www\framework\web\CBaseController.php(127): require('W:\home\lovehat...')
#3 W:\home\lovehatmy\www\framework\web\CBaseController.php(96): CBaseController->renderInternal('W:\home\lovehat...', Array, true)
#4 W:\home\lovehatmy\www\framework\web\CController.php(870): CBaseController->renderFile('W:\home\lovehat...', Array, true)
#5 W:\home\lovehatmy\www\framework\web\CController.php(783): CController->renderPartial('users_list', Array, true)
#6 W:\home\lovehatmy\www\protected\controllers\UserController.php(12): CController->render('users_list', Array)
#7 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionIndex()
#8 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#9 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#10 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('')
#12 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user')
#13 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/index.php?r=user
HTTP_REFERER=http://lovehatmy/index.php?r=user/signup
---
2012/07/24 12:06:42 [error] [exception.CException] exception 'CException' with message 'The "filter" property must be specified with a valid callback.' in W:\home\lovehatmy\www\framework\validators\CFilterValidator.php:46
Stack trace:
#0 W:\home\lovehatmy\www\framework\validators\CValidator.php(197): CFilterValidator->validateAttribute(Object(User), 'email')
#1 W:\home\lovehatmy\www\framework\base\CModel.php(158): CValidator->validate(Object(User), NULL)
#2 W:\home\lovehatmy\www\protected\controllers\UserController.php(28): CModel->validate()
#3 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#4 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#8 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#9 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user/signup
---
2012/07/24 12:26:37 [error] [exception.CException] exception 'CException' with message 'UserController and its behaviors do not have a method or closure named "getModels".' in W:\home\lovehatmy\www\framework\base\CComponent.php:266
Stack trace:
#0 [internal function]: CComponent->__call('getModels', Array)
#1 W:\home\lovehatmy\www\framework\web\form\CForm.php(205): UserController->getModels()
#2 W:\home\lovehatmy\www\protected\controllers\UserController.php(28): CForm->validate()
#3 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#4 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#8 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#9 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user/signup
---
2012/07/24 12:26:59 [error] [exception.CException] exception 'CException' with message 'The "filter" property must be specified with a valid callback.' in W:\home\lovehatmy\www\framework\validators\CFilterValidator.php:46
Stack trace:
#0 W:\home\lovehatmy\www\framework\validators\CValidator.php(197): CFilterValidator->validateAttribute(Object(User), 'email')
#1 W:\home\lovehatmy\www\framework\base\CModel.php(158): CValidator->validate(Object(User), NULL)
#2 W:\home\lovehatmy\www\protected\controllers\UserController.php(28): CModel->validate()
#3 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#4 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#8 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#9 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user/signup
---
2012/07/24 12:28:02 [error] [exception.CException] exception 'CException' with message 'The "filter" property must be specified with a valid callback.' in W:\home\lovehatmy\www\framework\validators\CFilterValidator.php:46
Stack trace:
#0 W:\home\lovehatmy\www\framework\validators\CValidator.php(197): CFilterValidator->validateAttribute(Object(User), 'email')
#1 W:\home\lovehatmy\www\framework\base\CModel.php(158): CValidator->validate(Object(User), 'user')
#2 W:\home\lovehatmy\www\protected\controllers\UserController.php(28): CModel->validate('user', true)
#3 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#4 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#8 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#9 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user/signup
---
2012/07/24 12:41:53 [error] [exception.CException] exception 'CException' with message 'The "filter" property must be specified with a valid callback.' in W:\home\lovehatmy\www\framework\validators\CFilterValidator.php:46
Stack trace:
#0 W:\home\lovehatmy\www\framework\validators\CValidator.php(197): CFilterValidator->validateAttribute(Object(User), 'email')
#1 W:\home\lovehatmy\www\framework\base\CModel.php(158): CValidator->validate(Object(User), NULL)
#2 W:\home\lovehatmy\www\protected\controllers\UserController.php(28): CModel->validate()
#3 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#4 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#8 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#9 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user/signup
---
2012/07/24 12:51:06 [error] [exception.CException] exception 'CException' with message 'The "filter" property must be specified with a valid callback.' in W:\home\lovehatmy\www\framework\validators\CFilterValidator.php:46
Stack trace:
#0 W:\home\lovehatmy\www\framework\validators\CValidator.php(197): CFilterValidator->validateAttribute(Object(User), 'email')
#1 W:\home\lovehatmy\www\framework\base\CModel.php(158): CValidator->validate(Object(User), NULL)
#2 W:\home\lovehatmy\www\protected\controllers\UserController.php(28): CModel->validate()
#3 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#4 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#8 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#9 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user/signup
---
2012/07/24 12:52:01 [error] [exception.CException] exception 'CException' with message 'The "filter" property must be specified with a valid callback.' in W:\home\lovehatmy\www\framework\validators\CFilterValidator.php:46
Stack trace:
#0 W:\home\lovehatmy\www\framework\validators\CValidator.php(197): CFilterValidator->validateAttribute(Object(User), 'email')
#1 W:\home\lovehatmy\www\framework\base\CModel.php(158): CValidator->validate(Object(User), NULL)
#2 W:\home\lovehatmy\www\protected\controllers\UserController.php(28): CModel->validate()
#3 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#4 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#8 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#9 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user/signup
---
2012/07/24 12:53:28 [error] [exception.CException] exception 'CException' with message 'User has an invalid validation rule. The rule must specify attributes to be validated and the validator name.' in W:\home\lovehatmy\www\framework\base\CModel.php:287
Stack trace:
#0 W:\home\lovehatmy\www\framework\base\CModel.php(259): CModel->createValidators()
#1 W:\home\lovehatmy\www\framework\base\CModel.php(544): CModel->getValidators()
#2 W:\home\lovehatmy\www\framework\base\CModel.php(466): CModel->getSafeAttributeNames()
#3 W:\home\lovehatmy\www\framework\base\CComponent.php(153): CModel->setAttributes(Array)
#4 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(160): CComponent->__set('attributes', Array)
#5 W:\home\lovehatmy\www\protected\controllers\UserController.php(25): CActiveRecord->__set('attributes', Array)
#6 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#7 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#8 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#9 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#11 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#12 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#13 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user/signup
---
2012/07/24 13:47:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "s".' in W:\home\lovehatmy\www\framework\web\CController.php:484
Stack trace:
#0 W:\home\lovehatmy\www\framework\web\CController.php(271): CController->missingAction('s')
#1 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('s')
#2 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/s')
#3 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=user/s
---
2012/07/24 13:47:56 [error] [exception.CException] exception 'CException' with message 'Property "User.text" is not defined.' in W:\home\lovehatmy\www\framework\base\CComponent.php:131
Stack trace:
#0 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(144): CComponent->__get('text')
#1 W:\home\lovehatmy\www\framework\web\helpers\CHtml.php(2044): CActiveRecord->__get('text')
#2 W:\home\lovehatmy\www\framework\web\helpers\CHtml.php(1258): CHtml::resolveValue(Object(User), 'text')
#3 W:\home\lovehatmy\www\protected\views\user\form_signup.php(28): CHtml::activeTextArea(Object(User), 'text')
#4 W:\home\lovehatmy\www\framework\web\CBaseController.php(127): require('W:\home\lovehat...')
#5 W:\home\lovehatmy\www\framework\web\CBaseController.php(96): CBaseController->renderInternal('W:\home\lovehat...', Array, true)
#6 W:\home\lovehatmy\www\framework\web\CController.php(870): CBaseController->renderFile('W:\home\lovehat...', Array, true)
#7 W:\home\lovehatmy\www\framework\web\CController.php(783): CController->renderPartial('form_signup', Array, true)
#8 W:\home\lovehatmy\www\protected\controllers\UserController.php(40): CController->render('form_signup', Array)
#9 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#10 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#11 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#12 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#14 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#15 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#16 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user
---
2012/07/24 15:00:03 [error] [exception.CException] exception 'CException' with message 'User has an invalid validation rule. The rule must specify attributes to be validated and the validator name.' in W:\home\lovehatmy\www\framework\base\CModel.php:287
Stack trace:
#0 W:\home\lovehatmy\www\framework\base\CModel.php(259): CModel->createValidators()
#1 W:\home\lovehatmy\www\framework\web\helpers\CHtml.php(1758): CModel->getValidators('login')
#2 W:\home\lovehatmy\www\framework\web\helpers\CHtml.php(1205): CHtml::activeInputField('text', Object(User), 'login', Array)
#3 W:\home\lovehatmy\www\protected\views\user\form_signup.php(8): CHtml::activeTextField(Object(User), 'login')
#4 W:\home\lovehatmy\www\framework\web\CBaseController.php(127): require('W:\home\lovehat...')
#5 W:\home\lovehatmy\www\framework\web\CBaseController.php(96): CBaseController->renderInternal('W:\home\lovehat...', Array, true)
#6 W:\home\lovehatmy\www\framework\web\CController.php(870): CBaseController->renderFile('W:\home\lovehat...', Array, true)
#7 W:\home\lovehatmy\www\framework\web\CController.php(783): CController->renderPartial('form_signup', Array, true)
#8 W:\home\lovehatmy\www\protected\controllers\UserController.php(40): CController->render('form_signup', Array)
#9 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#10 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#11 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#12 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#14 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#15 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#16 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user
---
2012/07/24 21:40:20 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDefaultValueValidator.format".' in W:\home\lovehatmy\www\framework\base\CComponent.php:174
Stack trace:
#0 W:\home\lovehatmy\www\framework\validators\CValidator.php(174): CComponent->__set('format', 'MM/dd/yyyy')
#1 W:\home\lovehatmy\www\framework\base\CModel.php(285): CValidator::createValidator('default', Object(User), 'birth_day', Array)
#2 W:\home\lovehatmy\www\framework\base\CModel.php(259): CModel->createValidators()
#3 W:\home\lovehatmy\www\framework\web\helpers\CHtml.php(1758): CModel->getValidators('login')
#4 W:\home\lovehatmy\www\framework\web\helpers\CHtml.php(1205): CHtml::activeInputField('text', Object(User), 'login', Array)
#5 W:\home\lovehatmy\www\protected\views\user\form_signup.php(8): CHtml::activeTextField(Object(User), 'login')
#6 W:\home\lovehatmy\www\framework\web\CBaseController.php(127): require('W:\home\lovehat...')
#7 W:\home\lovehatmy\www\framework\web\CBaseController.php(96): CBaseController->renderInternal('W:\home\lovehat...', Array, true)
#8 W:\home\lovehatmy\www\framework\web\CController.php(870): CBaseController->renderFile('W:\home\lovehat...', Array, true)
#9 W:\home\lovehatmy\www\framework\web\CController.php(783): CController->renderPartial('form_signup', Array, true)
#10 W:\home\lovehatmy\www\protected\controllers\UserController.php(40): CController->render('form_signup', Array)
#11 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#12 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#13 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#14 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#16 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#17 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#18 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/index.php?r=user/signup
HTTP_REFERER=http://lovehatmy/index.php?r=user
---
2012/07/24 21:41:00 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDefaultValueValidator.date".' in W:\home\lovehatmy\www\framework\base\CComponent.php:174
Stack trace:
#0 W:\home\lovehatmy\www\framework\validators\CValidator.php(174): CComponent->__set('date', 'MM/dd/yyyy')
#1 W:\home\lovehatmy\www\framework\base\CModel.php(285): CValidator::createValidator('default', Object(User), 'birth_day', Array)
#2 W:\home\lovehatmy\www\framework\base\CModel.php(259): CModel->createValidators()
#3 W:\home\lovehatmy\www\framework\web\helpers\CHtml.php(1758): CModel->getValidators('login')
#4 W:\home\lovehatmy\www\framework\web\helpers\CHtml.php(1205): CHtml::activeInputField('text', Object(User), 'login', Array)
#5 W:\home\lovehatmy\www\protected\views\user\form_signup.php(8): CHtml::activeTextField(Object(User), 'login')
#6 W:\home\lovehatmy\www\framework\web\CBaseController.php(127): require('W:\home\lovehat...')
#7 W:\home\lovehatmy\www\framework\web\CBaseController.php(96): CBaseController->renderInternal('W:\home\lovehat...', Array, true)
#8 W:\home\lovehatmy\www\framework\web\CController.php(870): CBaseController->renderFile('W:\home\lovehat...', Array, true)
#9 W:\home\lovehatmy\www\framework\web\CController.php(783): CController->renderPartial('form_signup', Array, true)
#10 W:\home\lovehatmy\www\protected\controllers\UserController.php(40): CController->render('form_signup', Array)
#11 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#12 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#13 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#14 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#16 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#17 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#18 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/index.php?r=user/signup
---
2012/07/24 21:59:01 [error] [exception.CException] exception 'CException' with message 'В классе User и его поведениях не найден метод или замыкание с именем "error".' in W:\home\lovehatmy\www\framework\base\CComponent.php:266
Stack trace:
#0 W:\home\lovehatmy\www\framework\db\ar\CActiveRecord.php(226): CComponent->__call('error', Array)
#1 [internal function]: CActiveRecord->__call('error', Array)
#2 W:\home\lovehatmy\www\protected\views\user\form_signup.php(24): User->error(Object(User), 'email')
#3 W:\home\lovehatmy\www\framework\web\CBaseController.php(127): require('W:\home\lovehat...')
#4 W:\home\lovehatmy\www\framework\web\CBaseController.php(96): CBaseController->renderInternal('W:\home\lovehat...', Array, true)
#5 W:\home\lovehatmy\www\framework\web\CController.php(870): CBaseController->renderFile('W:\home\lovehat...', Array, true)
#6 W:\home\lovehatmy\www\framework\web\CController.php(783): CController->renderPartial('form_signup', Array, true)
#7 W:\home\lovehatmy\www\protected\controllers\UserController.php(40): CController->render('form_signup', Array)
#8 W:\home\lovehatmy\www\framework\web\actions\CInlineAction.php(50): UserController->actionSignup()
#9 W:\home\lovehatmy\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 W:\home\lovehatmy\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 W:\home\lovehatmy\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 W:\home\lovehatmy\www\framework\web\CWebApplication.php(276): CController->run('signup')
#13 W:\home\lovehatmy\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/signup')
#14 W:\home\lovehatmy\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 W:\home\lovehatmy\www\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=user/signup
---
